api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSourceInterceptorContext.java

```diff
@@ -0,0 +1,9 @@
+package org.eclipse.microprofile.config.spi;
+
+public interface ConfigSourceInterceptorContext {
+    String getKey();
+
+    Class getConfigSource();
```

Comment:
Apart from the raw type usage - we want something more here, something that allows us to give the user some information about the configuration property.  But it should probably only be on `ConfigValue` because the context should be independent of any single configuration source.